git_machete/annotation.py

```diff
@@ -9,15 +9,17 @@ def __init__(self, annotation: str):
         self.__push_text: str = ''
```

Comment:
If you could also bump version in `git_machete/__init__.py` and add an entry in `RELEASE_NOTES.md`, crediting yourself (see the previous entries)